### PR TITLE
Correct CSS class name of MediaLibraryPickerField field

### PR DESCRIPTION
**Pull request by yw80**
Field CSS class name should use the technical name, rather than the display name.

### CODE REVIEW

File: src/Orchard.Web/Modules/Orchard.MediaLibrary/Views/Fields/MediaLibraryPicker.Summary.cshtml [hunk 1]
```diff
@@ -5,11 +5,12 @@
 @{
     var field = (MediaLibraryPickerField) Model.ContentField;
     string name = field.DisplayName;
```
Comment (sebastienros) [inline src/Orchard.Web/Modules/Orchard.MediaLibrary/Views/Fields/MediaLibraryPicker.Summary.cshtml]: change it to `displayName`

File: src/Orchard.Web/Modules/Orchard.MediaLibrary/Views/Fields/MediaLibraryPicker.Summary.cshtml [hunk 2]
```diff
@@ -5,11 +5,12 @@
...
+    string techName = field.Name;
```
Comment (sebastienros) [inline src/Orchard.Web/Modules/Orchard.MediaLibrary/Views/Fields/MediaLibraryPicker.Summary.cshtml]: change it to `name`

**Comment by yw80** [comment]
The new changes have been committed.

**Comment by yw80** [comment]
@sebastienros
The file has been modified in accordance with your revised.

**Pull request merged**